.idea/.name

```diff
@@ -0,0 +1 @@
+training
```

Comment:
All .idea should not be in the remote repository

Reply:
All .idea it is all out